script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -684,7 +684,7 @@ But you see, no matter how far removed we become from that foundation,
 [sha:542ad5b18cd1e6178e61023459b606acdeeeca3d]{
 -- Page 478, Offset 17757.
 -- 「全ての根源、全ての始まりと終わりを記録したレコード。
-"The root of all things. A record of the beginning and the end of everything.
+"The root of all things: a record of the beginning, and the end of everything.
```

Comment:
I interpreted this as a record that contains both the beginning and the end, the comma kinda makes it feel a bit more separate?